packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -746,6 +772,14 @@ function ResizableTableColumnHeader(props) {
   }, []);
 
   let showResizer = !isEmpty && ((headerRowHovered && getInteractionModality() !== 'keyboard') || resizingColumn != null);
+  let alignment = 'start';
+  let menuAlign = 'start' as 'start' | 'end';
+  if (columnProps.align === 'center' || column.colspan > 1) {
```

Comment:
Unrelated to the PR, but this reminded me that nested columns were a thing so I tried adding resizing to those stories:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/9661127/212400359-ba43c1f8-c767-4eee-ab45-360c11d9a4be.png">

Noted that I couldn't resize the parent columns, not sure we want to support that right now/may need to discuss the behavior there. Do we wanna disable resizing for columns with children for now? Happy to open a PR if so.

Reply:
Yeah, I have not done anything with parent columns, I'm not sure how that would work. Maybe it'd be good to disable it from being an option to even set that prop somehow...

Reply:
How about we add a warning in TableView if `column.hasChildNodes` + `allowsResizing` is provided. We could then add a couple checks in other places to prevent a column from rendering as a resizable column if it has childNodes

Reply:
I guess, do actions still matter for combined columns?
I'll merge this one and make a follow up to discuss